### DISCUSSION TITLE
Quiz Data Wrangling and Visualization no 4

### DISCUSSION THREAD

**Discussion by Zahraa02** [teamalgoritma/community#934] [Q&A] [answered]
Selamat malam, saya sudah menyelesaikan quiznya, tetapi saya lihat bahwa saya salah di no 4. Berdasarkan hasil yang saya peroleh, diketahui bahwa 'May to June' merupakan penurunan pendapatan yang paling signifikan,  akan tetapi jawabannya salah. Kira-kira coding saya salah dibagian mana yaa?
![image](https://github.com/user-attachments/assets/7fc626e3-aff7-444d-92c1-bc31c753b8d4)
![image](https://github.com/user-attachments/assets/ce1b1898-8a5d-4ba2-a1df-1583d2dde018)
![image](https://github.com/user-attachments/assets/f2840447-87a3-408c-a7c7-738556e0e238)

**Comment by saskia-dwi-ulfah** [comment, marked as answer]
Dear Bu @Zahraa02 
Terima kasih sudah bertanya melalui Github Community Algoritma.

Terkait coding-an ibu sudah benar bu dan plot yang dihasilkan juga sudah sesuai. Selain itu, jawaban ibu juga sudah benar. Akan tetapi, terdapat kesalahan pada kunci pada Algoritma Online sehingga jawaban ibu dinilai tidak tepat. 

Untuk hal ini sudah kami sesuaikan bu dengan kunci yang benar. Kami juga mengucapkan terima kasih karena ibu telah membantu kami untuk men-tracing kesalahan tersebut dan mohon maaf atas ketidaknyamanan yang ditimbulkan 🙏

Best Regards,
Team Algoritma

**Reply by Zahraa02** [reply]
Ooh okaay, terima kasih banyak🙏
